.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,31 @@
+<!-- markdownlint-disable MD041 first-line-heading/first-line-h1 -->
+
 ## Notes for Reviewer
 <!-- Items in addition to the checklist below that the reviewer should look for -->
 
+## Prevent CI from running on each commit while the work is still in progress
+
+* Set the PR to a draft status, e.g. by the `Convert to draft` link, to indicate
+that the work is not yet ready for review
+* Once the PR is ready for review, press the `Ready for Review` button and push
+a final commit to trigger the CI
+
 ## Pre-Review Checklist for the PR Author
 
-1. [ ] Add sensible notes for the reviewer
-1. [ ] PR title is short, expressive and meaningful
-1. [ ] Relevant issues are linked in the [References](#references) section
-1. [ ] Every source code file has a copyright header with `SPDX-License-Identifier: Apache-2.0 OR MIT`
-1. [ ] Branch follows the naming format (`iox2-123-introduce-posix-ipc-example`)
-1. [ ] Commits messages are according to this [guideline][commit-guidelines]
-    - [ ] Commit messages have the issue ID (`[#123] Add posix ipc example`)
-    - [ ] Commit author matches [Eclipse Contributor Agreement][eca] (and ECA is signed)
-1. [ ] Tests follow the [best practice for testing][testing]
-1. [ ] Changelog updated [in the unreleased section][changelog] including API breaking changes
-1. [ ] Assign PR to reviewer
-1. [ ] All checks have passed (except `task-list-completed`)
+* [ ] Add sensible notes for the reviewer
+* [ ] PR title is short, expressive and meaningful
+* [ ] Relevant issues are linked in the [References](#references) section
+* [ ] Every source code file has a copyright header with
+`SPDX-License-Identifier: Apache-2.0 OR MIT`
+* [ ] Branch follows the naming format (`iox2-123-introduce-posix-ipc-example`)
+* [ ] Commits messages are according to this [guideline][commit-guidelines]
+    * [ ] Commit messages have the issue ID (`[#123] Add posix ipc example`)
+    * [ ] Commit author matches [Eclipse Contributor Agreement][eca](and ECA is signed)
+* [ ] Tests follow the [best practice for testing][testing]
+* [ ] Changelog updated [in the unreleased section][changelog] including API
+breaking changes
+* [ ] Assign PR to reviewer
```

Comment:
I'd like to remove the 80 character limitation. This totally breaks the reading flow

Reply:
Shall we go for 110?